tensorflow/python/training/adam.py

```diff
@@ -29,7 +29,7 @@
 
 
 class AdamOptimizer(optimizer.Optimizer):
-  """Optimizer that implements the Adam algorithm.
+  """Optimizer that implements the Adam algorithm (http://arxiv.org/abs/1412.6980).
```

Comment:
Sorry didn't see it. Should the ref be in init or in the class docstring?

Elsewhere, the arxiv reference is in the class docstring:

class GRUCell(RNNCell):"""Gated Recurrent Unit cell (cf.
http://arxiv.org/abs/1406.1078)."""

On Sat, Jan 2, 2016 at 7:30 PM, rafaljozefowicz <notifications@github.com>
wrote:

> In tensorflow/python/training/adam.py
> <https://github.com/tensorflow/tensorflow/pull/670#discussion_r48689729>:
>
> > @@ -29,7 +29,7 @@
> >
> >
> >  class AdamOptimizer(optimizer.Optimizer):
> > -  """Optimizer that implements the Adam algorithm.
> > +  """Optimizer that implements the Adam algorithm (http://arxiv.org/abs/1412.6980).
>
> Line 41 references the paper.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/tensorflow/tensorflow/pull/670/files#r48689729>.
>